File: BSIGeneralAffairBO/Asset.vb
Public Class Asset
    Public Property AssetID As Integer?
    Public Property BrandID As Short?
    Public Property AssetCategoryID As Short?
    Public Property AssetFactoryNumber As String
    Public Property AssetNumber As String
    Public Property AsssetName As String
    Public Property AssetCost As Decimal?
    Public Property AssetProcurementDate As DateTime?
    Public Property AssetFlagActive As Boolean?
    Public Property AssetCondition As String
    Public Property CreatedAt As DateTime?
    Public Property UpdatedAt As DateTime?
    Public Property Category As AssetCategory
    Public Property UserAsset As List(Of AssetUser)
    Public Property Brand As Brand


End Class

File: BSIGeneralAffairBO-C/Asset.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Text;

namespace BSIGeneralAffairBO_C
{
    public class Asset
    {
        public int? AssetID { get; set; }
        public short? BrandID { get; set; }
        public short? AssetCategoryID { get; set; }
        public string AssetFactoryNumber { get; set; }
        public string AssetNumber { get; set; }
        public string AssetName { get; set; }
        public decimal? AssetCost { get; set; }
        public string AssetProcurementDate { get; set; }
        public bool? AssetFlagActive { get; set; }
        public string AssetCondition { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
        public AssetCategory Category { get; set; }
        public List<AssetUser> UserAsset { get; set; }
        public Brand Brand { get; set; }
    }
}
